src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php

```diff
@@ -19,17 +22,39 @@
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  * @author Robin Chalas <robin.chalas@gmail.com>
  */
-class YamlLintCommand extends BaseLintCommand
+class YamlLintCommand extends Command
 {
+    private $command;
+
     /**
      * {@inheritdoc}
      */
     protected function configure()
     {
-        parent::configure();
+        $this->setName('lint:yaml');
+
+        if (!$this->isEnabled() && null !== $this->command) {
```

Comment:
Do we need this? `configure()` should never be called if the command is not enabled.